index.html

```diff
@@ -214,12 +219,12 @@ <h1 color="#FFFFFF" class="css-1rvqtgb">City of Boulder Careers</h1>
                                             <div class="css-qiqmbt">
                                                 <div class="css-b3pn3b">
                                                     <div class="css-b3pn3b">
-                                                        <h3><a aria-current="false" data-automation-id="jobTitle"
+                                                        <h2><a aria-current="false" data-automation-id="jobTitle"
```

Comment:
Document outline showed that there are <h3> tags which belong directly to an <h2>, this fixes that without compromising styles or layout

![Screen Shot 2024-11-23 at 11 23 47 PM](https://github.com/user-attachments/assets/78084160-4083-4284-8d8b-4218c94cfe36)